chaosazure/common/resources/graph.py

```diff
@@ -9,6 +9,7 @@
 
 from chaosazure import init_resource_graph_client
 from chaosazure.common.config import load_configuration
+import azure.mgmt.resourcegraph as arg
```

Comment:
Imports should follow an order such as:

```
standard lib imports

third party imports

package imports
```

Here, this import should be on line 8.